deploy/basic/terraform/atp.tf

```diff
@@ -9,7 +9,7 @@ resource "oci_database_autonomous_database" "mushop_autonomous_database" {
   compartment_id           = var.compartment_ocid
   cpu_core_count           = 1
   data_storage_size_in_tbs = 1
-  db_name                  = "${var.database_name}${random_id.mushop_id.dec}"
+  db_name                  = var.database_name
```

Comment:
MuShop basic doesn't need to random_id, as the number of Always Free ATPs are limited.

Reply:
And this was affecting the validation of number of characters max from the stack schema